### PR TITLE
Change out link in the welcome text for slack etiquette

### PR DESCRIPTION
**Pull request by peoray**
Fix issue #94.

Change the link for slack etiquette for general members from https://codebuddies.slack.com/files/U0G92F4J0/F8VRAGD0S/CodeBuddies_Slack_Etiquette_for_General_Members to https://codebuddies.org/slack

### CODE REVIEW

File: src/app/routes/endpoints/events/initial.js [hunk 1]
```diff
@@ -74,7 +74,7 @@ const message = {
...
...e &amp; fill in the information. All hangouts get automatically posted back to <https://codebuddies.slack.com/archives/C04AQ6GH0|#announcements>.\n\n\n:books:  Interested in going deep on topics like * Functional JS* or that special *Ruby* book or *Blockchain*? Start a <codebuddies.org/study-groups|StudyGroup>!  All study groups created on the website also get automatically posted back to <https://codebuddies.slack.com/archives/C04AQ6GH0|#announcements>.  Invite folks here on Slack to join in so that you can motivate each other to master your learning goal.\n\n\n:question:  If you have any questions about the community or the project or want to talk to an admin, feel free to ask about it in <https://codebuddies.slack.com/archives/C04BRN86J|#codebuddies-meta>. It's where discussions related to our community happen.\n\n\n:heavy_plus_sign:  If you'd like to add a new integration to the Slack, please ask in the <https://codebuddies.slack.com/archives/C04BRN86J|#codebuddies-meta> channel as well! Because CodeBuddies is on a free plan, the number of new Slack apps we can integrate is limited, so we would need to choose our integrations wisely.\n\n\n_Happy Learning!_ :wave::skin-tone-5:",
```
Comment (stain88) [inline src/app/routes/endpoints/events/initial.js]: Looks like a stray dot there: `https://codebuddies.org./slack`

**Review by stain88 — CHANGES_REQUESTED** [review]
Just one minor change

File: src/app/routes/endpoints/events/initial.js (hunk 1 above)
Comment (stain88) [reply src/app/routes/endpoints/events/initial.js]: and the pipe is missing between link and text

Comment (peoray) [reply src/app/routes/endpoints/events/initial.js]: Fixed, thank for spotting it out

**Review by stain88 — APPROVED** [review]

**Pull request merged**